docs/guides/how-to-reset-state.md

```diff
@@ -78,3 +72,4 @@ export const resetAllSlices = () => {
 
 - Basic: https://codesandbox.io/s/zustand-how-to-reset-state-basic-demo-rrqyon
 - Advanced: https://codesandbox.io/s/zustand-how-to-reset-state-advanced-demo-gtu0qe
+- Immer: https://codesandbox.io/s/how-to-reset-state-advance-immer-demo-nyet3f
```

Comment:
Here `useTuna` should be `create<StateTuna & ActionsTuna>()(immer(...` and not `create(immer<StateTuna & ActionsTuna>(...`

Reply:
Sure

Reply:
@devanshj now we are good

Reply:
Yep looks fine. Now this one change remains...

> And let's also change `create<State & Actions>(...` into `create<State & Action>()(...` in the first snippet of the document as I suggested earlier (can't comment on that line as it's not part of the change).